pandas/__init__.py

```diff
@@ -49,7 +49,15 @@
 from pandas.tools.merge import (merge, ordered_merge,
                                 merge_ordered, merge_asof)
 from pandas.tools.pivot import pivot_table, crosstab
-from pandas.tools.plotting import scatter_matrix, plot_params
+
+# deprecate tools.plotting, plot_params and scatter_matrix on the top namespace
+import pandas.tools.plotting
```

Comment:
yeah its probably fine. also need to move these in the test_api to the deprecated section. (also add on to the deprecated issue tracker)